intro-HPC/ch_troubleshooting.tex

```diff
@@ -91,6 +91,20 @@ \section{Issues connecting to login node}
             share your private key, they're called \emph{private} for a good reason.
 \end{enumerate}
 
+\ifwindows
+
+If you're using PuTTY and get this errormessage:
+
+\begin{prompt}
+server unexpectedly closed network connection
+\end{prompt}
+
+it can be because your PuTTY version is too old. Make sure you're using at least
+version 0.70. If your PuTTY version is too old, you can uninstall it and then
```

Comment:
Don't mention a specific version (or certainly not without mentioning why).

This gets outdated too easily, just tell people to make sure they have the latest version of PuTTY in case they run into problems.

Reply:
Is uninstall really needed? Wouldn't installing the latest overwrite?

Also, they don't need to *install*, strictly speaking (just using the `.exe` works too), so maybe reword?